### ISSUE TITLE
Aborted at 1515694750 (unix time) try "date -d @1515694750" if you are using GNU date

### ISSUE THREAD

**Issue by harold-yh** [PaddlePaddle/DeepSpeech#123]
系统环境：Ubuntu 16.04,使用的两块GTX 1080TI显卡，内存32G，SWAP：800G

参照aishell脚本训练自己的语音和标注数据，训练结束时报错：
...................................................................................................
Pass: 18, Batch: 100, TrainCost: 70.923850
............................................................................
------- Time: 759 sec,  Pass: 18, ValidationCost: 131.906596488
...................................................................................................
Pass: 19, Batch: 100, TrainCost: 70.486890
............................................................................
------- Time: 754 sec,  Pass: 19, ValidationCost: 106.161569876
*** Aborted at 1515694750 (unix time) try "date -d @1515694750" if you are using GNU date ***

使用的参数如下：
-----------  Configuration Arguments -----------
augment_conf_path: conf/augmentation.config
batch_size: 8
dev_manifest: data/numbers/manifest.dev
init_model_path: checkpoints/numbers/params.pass-19.tar.gz
is_local: 1
learning_rate: 1e-06
max_duration: 27.0
mean_std_path: data/numbers/mean_std.npz
min_duration: 0.0
num_conv_layers: 2
num_iter_print: 100
num_passes: 20
num_proc_data: 5
num_rnn_layers: 3
output_model_dir: ./checkpoints/numbers
rnn_layer_size: 2048
share_rnn_weights: 1
shuffle_method: batch_shuffle_clipped
specgram_type: linear
test_off: 0
train_manifest: data/numbers/manifest.train
trainer_count: 2
use_gpu: 1
use_gru: 1
use_sortagrad: 1
vocab_path: data/numbers/vocab.txt
------------------------------------------------

**Comment by kuke** [comment]
看起来是环境问题，可能和python或shell有关

**Comment by harold-yh** [comment]
用docker训练也是一样报这个错，训练完成时报的。
paddle最新GPU版本。
*** Aborted at 1516100831 (unix time) try "date -d @1516100831" if you are using GNU date ***
PC: @                0x0 (unknown)
*** Aborted at 1516100834 (unix time) try "date -d @1516100834" if you are using GNU date ***
PC: @                0x0 (unknown)
*** Aborted at 1516100838 (unix time) try "date -d @1516100838" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGTERM (@0x23f3) received by PID 12032 (TID 0x7fad3a742700) from PID 9203; stack trace: ***
    @     0x7fad3a31d390 (unknown)
    @     0x7fad3a31c76d __GI_accept
    @           0x597362 (unknown)
    @           0x4c96e0 PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c2705 PyEval_EvalCodeEx
    @           0x4de858 (unknown)
    @           0x4b0c93 PyObject_Call
    @           0x4c6ef6 PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c2705 PyEval_EvalCodeEx
    @           0x4de69e (unknown)
    @           0x4b0c93 PyObject_Call
    @           0x4f452e (unknown)
    @           0x4b0c93 PyObject_Call
    @           0x4f42a7 (unknown)
    @           0x4b669c (unknown)
    @           0x4b0c93 PyObject_Call
    @           0x4c9f9f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c2705 PyEval_EvalCodeEx
    @           0x4ca088 PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4ddd6a (unknown)
    @           0x4c4bdc PyEval_EvalFrameEx
    @           0x4ddd6a (unknown)
    @           0x4c4bdc PyEval_EvalFrameEx
    @           0x4ddd6a (unknown)
*** Aborted at 1516100840 (unix time) try "date -d @1516100840" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGTERM (@0x23f3) received by PID 9356 (TID 0x7fad3a742700) from PID 9203; stack trace: ***
    @     0x7fad3a31d390 (unknown)
    @     0x7fad3a31c76d __GI_accept
    @           0x597362 (unknown)
    @           0x4c96e0 PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c2705 PyEval_EvalCodeEx
    @           0x4de858 (unknown)
    @           0x4b0c93 PyObject_Call
    @           0x4c6ef6 PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c2705 PyEval_EvalCodeEx
    @           0x4de69e (unknown)
    @           0x4b0c93 PyObject_Call
    @           0x4f452e (unknown)
    @           0x4b0c93 PyObject_Call
    @           0x4f42a7 (unknown)
    @           0x4b669c (unknown)
    @           0x4b0c93 PyObject_Call
    @           0x4c9f9f PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4c2705 PyEval_EvalCodeEx
    @           0x4ca088 PyEval_EvalFrameEx
    @           0x4c9d7f PyEval_EvalFrameEx
    @           0x4ddd6a (unknown)
    @           0x4c4bdc PyEval_EvalFrameEx
    @           0x4ddd6a (unknown)
    @           0x4c4bdc PyEval_EvalFrameEx
    @           0x4ddd6a (unknown)
*** Aborted at 1516100842 (unix time) try "date -d @1516100842" if you are using GNU date ***
PC: @                0x0 (unknown)

**Comment by s11e22** [comment]
Hi there, I am facing the Aborted situation as same as yours. Have you got the solution?
My environment setting is in the docker  using image - paddlepaddle/deep_speech:latest-gpu.
1 1080ti card and 11GB RAM 
I follow the guide cd into the folder /Deepspeech/example/tiny
 running `sh run_data.sh` things seems fine here and then next step
`sh run_train.sh` looks good and the out is successfully print on the screen
BUT when I adjust the Configuration Arguments below
--num_passes=20 \  (default)
--num_passes=50 \ (I add more passes)
Aborted message appear at the end of trainning.
like this, just the same as @harold-yh paste up there


------- Time: 3 sec,  Pass: 48, ValidationCost: 136.356533051
...
------- Time: 3 sec,  Pass: 49, ValidationCost: 134.335668564
*** Aborted at 1517215622 (unix time) try "date -d @1517215622" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGTERM (@0x297b) received by PID 18396 (TID 0x7f2c8c464700) from PID 10619; stack trace: ***
    @     0x7f2c8c040390 (unknown)
    @     0x7f2c8c03f76d __GI_accept
    @           0x598072 (unknown)
    @           0x4c9780 PyEval_EvalFrameEx
    @           0x4c9d8f PyEval_EvalFrameEx
    @           0x4c9d8f PyEval_EvalFrameEx
    @           0x4c9d8f PyEval_EvalFrameEx
    @           0x4c9d8f PyEval_EvalFrameEx
    @           0x4c2765 PyEval_EvalCodeEx
    @           0x4de8b8 (unknown)
    @           0x4b0cb3 PyObject_Call
    @           0x4c6ad1 PyEval_EvalFrameEx
    @           0x4c9d8f PyEval_EvalFrameEx
    @           0x4c9d8f PyEval_EvalFrameEx
    @           0x4c2765 PyEval_EvalCodeEx
    @           0x4de6fe (unknown)
    @           0x4b0cb3 PyObject_Call
    @           0x4f492e (unknown)
    @           0x4b0cb3 PyObject_Call
    @           0x4f46a7 (unknown)
    @           0x4b670c (unknown)
    @           0x4b0cb3 PyObject_Call
    @           0x4c9faf PyEval_EvalFrameEx
    @           0x4c9d8f PyEval_EvalFrameEx
    @           0x4c2765 PyEval_EvalCodeEx
    @           0x4ca099 PyEval_EvalFrameEx
    @           0x4c9d8f PyEval_EvalFrameEx
    @           0x4dddca (unknown)
    @           0x4c4c6f PyEval_EvalFrameEx
    @           0x4dddca (unknown)
    @           0x4c4c6f PyEval_EvalFrameEx
    @           0x4dddca (unknown)
*** Aborted at 1517215622 (unix time) try "date -d @1517215622" if you are using GNU date ***
*** Aborted at 1517215623 (unix time) try "date -d @1517215623" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGTERM (@0x297b) received by PID 8384 (TID 0x7f2c8c464700) from PID 10619; stack trace: ***
*** Aborted at 1517215623 (unix time) try "date -d @1517215623" if you are using GNU date ***
PC: @                0x0 (unknown)



Anyone got some advice for this problem?
Thanks a lot.

**Comment by iamadog3333** [comment]
I met the same problem.
some people believe its because of not enough memory, some people think that because the code use a 'unix time' as a random seed when do shuffle work.
But how to fix it?
不知如何修复。
-------
 *** Aborted at 1520768519 (unix time) try "date -d @1520768519" if you are using GN     U date ***
3510 PC: @                0x0 (unknown)
3511 *** SIGFPE (@0x7f10ff0f4c47) received by PID 29777 (TID 0x7f1197d32700) from PID 18     446744073693776967; stack trace: ***
3512     @     0x7f1197909160 (unknown)
3513     @     0x7f10ff0f4c47 sgemm_kernel_SANDYBRIDGE

**Comment by pkuyym** [comment]
@iamadog3333 Hi,  `SIGFPE` may be caused by unstable training, please carefully tune your hyper parameters including learning rate/batch size etc.

**Comment by xguo-prestolabs** [comment]
PC: @ 0x0 (unknown) 是什么意思？
